src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let discount = 0;
+  let sum = 0;
+  const price = 40;
+
+  if (days > 2 && days < 7) {
+    discount = 20;
+  }
+
+  if (days >= 7) {
+    discount = 50;
+  }
```

Comment:
what a magic numbers 🪄
lets create variables and then use them
it is better for managing them from one place and for understanding what do they mean